index.js

```diff
@@ -175,6 +180,13 @@ function getErrorMessage (err, status, env) {
     // use err.stack, which typically includes err.message
     msg = err.stack
 
+    // if error cause included
```

Comment:
Does it handle nested cause? What if the cause itself has another cause? Also, perhaps it should append the stack trace of the error itself, regardless of whether the cause exists or not?